### PR TITLE
FR-1733: adding end states for general order

### PR DESCRIPTION
**Pull request by ocarleton**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/FR-1733

### Change description ###

adds end states to general order events

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
       "ccdUrl": "http://ccd-data-store-api-demo.service.core-compute-demo.internal"
     },
     "aat": {
-      "cosUrl": "http://finrem-cos-aat.service.core-compute-aat.internal",
+      "cosUrl": "http://finrem-cos-pr-390.service.core-compute-preview.internal",
```
Comment (ocarleton) [inline package.json:22]: to be reverted before merge

File: definitions/consented/json/CaseEvent.json [hunk 2]
```diff
@@ -367,7 +367,7 @@
     "Description": "Create General Order",
     "DisplayOrder": 14,
     "PreConditionState(s)": "*",
-    "PostConditionState": "*",
+    "PostConditionState": "orderMade",
```
Comment (HarryH96) [inline definitions/consented/json/CaseEvent.json:370]: What events can be triggered from orderMade or orderDrawn?
Do these need to be added to precondition state for the close event?

Comment (ocarleton) [reply definitions/consented/json/CaseEvent.json:370]: shouldn't need added to close as close event preconditions are
    "PreConditionState(s)": "*",
for contested and order made is in the consent order preconditions

from order drawn a court admin can..
image

File: definitions/contested/json/FixedLists/FixedLists.json [hunk 3]
```diff
@@ -1535,6 +1535,57 @@
...
+    "LiveFrom": "01/01/2017",
+    "ID": "FR_fl_GeneralOrderJudgeType",
+    "ListElementCode": "Her Honour Judge",
+    "ListElement": "Her Honour Judge"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "FR_fl_GeneralOrderJudgeType",
+    "ListElementCode": "Recorder",
+    "ListElement": "Recorder"
```
Comment (shaunganley) [inline definitions/contested/json/FixedLists/FixedLists.json:1587]: duplicate? https://github.com/hmcts/finrem-ccd-definitions/blob/af20c96325c3a2b72fd159176c5c30e0a4d912a6/definitions/contested/json/FixedLists/FixedLists.json#L1692

**Comment by ocarleton** [comment]
closing to make a clean PR without merges bringing extra

**Pull request closed without merging**